js/utils/__test__/array.split.test.js

```diff
@@ -12,6 +12,27 @@ function isThird(x) {
     return x % 3 === 0;
 }
 
+describe('test utilities', () => {
+    it('correctly checks if input for isEven() is an even number', () => {
+        expect(isEven(0)).toBe(true);
+        expect(isEven(1)).toBe(false);
+        expect(isEven(2)).toBe(true);
+    });
+
+    it('correctly checks if input for isOdd() is an odd number', () => {
+        expect(isOdd(0)).toBe(false);
+        expect(isOdd(1)).toBe(true);
+        expect(isOdd(2)).toBe(false);
+    });
+
+    it('correctly checks if input for isThird() is divisible by 3', () => {
+        expect(isThird(0)).toBe(true);
```

Comment:
Weet je zeker dat je bij 0 ook true wilt hebben? En niet `x > 0 && x % 3 === 0?`

Reply:
Ja, inderdaad. 0 is niet deelbaar door 3